src/fsharp/service/service.fs

```diff
@@ -3209,10 +3209,10 @@ type FSharpChecker(legacyReferenceResolver, projectCacheSize, keepAssemblyConten
         backgroundCompiler.KeepProjectAlive(options, userOpName)
 
     /// For a given script file, get the ProjectOptions implied by the #load closure
-    member ic.GetProjectOptionsFromScript(filename, sourceText, ?loadedTimeStamp, ?otherFlags, ?useFsiAuxLib, ?assumeDotNetFramework, ?extraProjectInfo: obj, ?optionsStamp: int64, ?userOpName: string) = 
+    member ic.GetProjectOptionsFromScript(filename, source, ?loadedTimeStamp, ?otherFlags, ?useFsiAuxLib, ?useSdkRefs, ?assumeDotNetFramework, ?extraProjectInfo: obj, ?optionsStamp: int64, ?userOpName: string) = 
```

Comment:
Why the switch from `sourceText` to `source`?  I thought the underlying type was `ISourceText` instead of `string`, now.

Reply:
Yes, the underlying type is `ISourceText` here. Why isn't this failing to compile? `source` isn't defined anywhere.

Reply:
That's just a merge change.  The code I started with called it source, someone renamed it to sourcetext in master, now my pr is trying to change it back.  I will use sourceText.  Note: the name doesn't actually impact the type, it's still going to be an ISourceText argument.

Reply:
Done